src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -18,7 +18,7 @@ export default class PhoneNumberWidget extends React.Component {
   handleChange = val => {
     let stripped;
     if (val) {
-      stripped = val.replace(/[ \-()x+]/g, '');
+      stripped = val.replace(/[ \-()x+~!@#$%^&*_=]/g, '');
```

Comment:
Would it be simpler to match on regex non-digit? \D

Reply:
I don't think that is best course of action because of general recommendation to not do much validation or modifying of phone numbers that are provided by a user.

Reply:
Updated, Only allows number or text (excluding the letter `x` )